### PR TITLE
net/gnrc_netif: clarify documentation of gnrc_netif_ipv6_addrs_get()

### PR DESCRIPTION
**Pull request by benpicco**
### Contribution description

Reformulate the documentation to make it clear that this returns the *size* of the addresses, not the number of addresses.

### Testing procedure

Just read and see if this is less easy to misunderstand.

### CODE REVIEW

**Comment by kaspar030** [comment]
LGTM. @miri64 what do you think?

File: sys/include/net/gnrc/netif.h [hunk 1]
```diff
@@ -392,8 +392,8 @@ gnrc_netif_t *gnrc_netif_get_by_pid(kernel_pid_t pid);
  *                      @p CONFIG_GNRC_NETIF_IPV6_ADDRS_NUMOF `* sizeof(ipv6_addr_t)
  *                      here (and have @p addrs of the according length).
  *
- * @return  Number of addresses in @p addrs times `sizeof(ipv6_addr_t)` on
- *          success (including 0).
+ * @return  Size of addresses in @p addrs (number of addresses times `sizeof(ipv6_addr_t)`)
```
Comment (miri64) [inline sys/include/net/gnrc/netif.h:395]:
```suggestion
 * @return  Size of the array of addresses in @p addrs (number of addresses times `sizeof(ipv6_addr_t)`)
```

? Size of addresses IMHO is ambiguous (could also just mean `sizeof(ipv6_addr_t)`)

Comment (miri64) [reply sys/include/net/gnrc/netif.h]: Though, on the other hand, "Size of the array of addresses" could also be misunderstood as `ARRAY_SIZE(addresses)` mhhhh :-/

Comment (miri64) [reply sys/include/net/gnrc/netif.h]: But maybe, with the clarifying statement in parentheses this is enough now.

File: sys/include/net/gnrc/netif.h [hunk 2]
```diff
@@ -392,8 +392,8 @@ gnrc_netif_t *gnrc_netif_get_by_pid(kernel_pid_t pid);
  *                      @p CONFIG_GNRC_NETIF_IPV6_ADDRS_NUMOF `* sizeof(ipv6_addr_t)
  *                      here (and have @p addrs of the according length).
  *
- * @return  Number of addresses in @p addrs times `sizeof(ipv6_addr_t)` on
- *          success (including 0).
+ * @return  Size of the array of addresses in @p addrs (number of addresses times `sizeof(ipv6_addr_t)`)
```
Comment (miri64) [inline sys/include/net/gnrc/netif.h:395]: image

**Review by miri64 — COMMENTED** [review]
Please squash after applying the line-length fix

**Review by miri64 — APPROVED** [review]

**Pull request merged**